### PR TITLE
13 ranking guesses

### PR DESCRIPTION
**Pull request by nac5504**
#13 

Would like some feedback on the getTextColor and UIColor extension, which is designed to scan the bottom 1/5 of the image, determine the average rgb color, then convert to grayscale and determine whether white or black text would appear better.

Also, the List component appears with too much grayspace at the top. Not sure how to get rid of this.

### CODE REVIEW

File: Guess-E/RankingGuessesView.swift [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                        VStack(alignment: .leading, spacing: 3) {
+                            Text("You")
+                                .bold()
+                            Text("\"This is what you told the AI to generate. It should be in quotes.\"")
+                            
+                            
+                        }.frame(alignment: .leading)
+                            .font(.system(size: 17))
+                            .padding()
+                            .foregroundColor(self.determineTextColor(image: imageName))
```
Comment (atlant04) [inline Guess-E/RankingGuessesView.swift:65]: This seems pretty complicated, I think just having a black linear gradient + white text would look just fine no matter the average color of the underlying photo. If it's black, the gradient would be barely visible and the text would look nice. If it's white, then the gradient will provide enough contrast for the text to be visible.

**Review by atlant04 — COMMENTED** [review]
might be a bit too complex, see the comments. Otherwise looks fine, pre-approving it now.

**Review by atlant04 — APPROVED** [review]

**Comment by atlant04** [comment]
yeah looks good. it's already approved, so you can merge whenever after the conflicts are resolved.

**Comment by nac5504** [comment]
I don't think I have permissions to choose which files are included in the merge.

**Comment by atlant04** [comment]
You should, any committed files will be merged unless some of those files have a merge conflict with main, in this case, it's the pbxproj. The easiest way to resolve it is to:
1. Pull the most recent main.
2. Switch to your branch (git switch 13-ranking-guesses)
3. rebase your branch onto main (git rebase main) 
4. this will generate a conflict in the pbxproj. open that file in VSCode (or other tools) and accept both changes from each branch
5. push your branch to the remote branch (git push -f origin 13-ranking-guesses). Note the -f flag for force push. 
6. At this point everything should be resolved and you can merge here

**Pull request merged**